### DISCUSSION TITLE
Faulty multi ranger deck?

### DISCUSSION THREAD

**Discussion by ugrm** [bitcraze/discussions#1413] [Q&A] [answered]
Hi all,

I tried to use the multi ranger deck and get some basic measurements like this (https://forum.bitcraze.io/viewtopic.php?t=5300). All the other plotter options work fine but it seems that the "range.back, range.front, range.left, range.right, range.up, range.zrange" options don't seem to work and stay constant regardless of the surroundings. 
Are there any tests I can run to make sure that the problem is caused by the multi-ranger deck? If there are any software changes I can make please let me know. 
This is what the screen looks like. 

![image](https://github.com/bitcraze/discussions/assets/145661518/664f0683-6f96-44fd-97b2-aeecff21a43c)

**Comment by ugrm** [comment]
I'm using Crazyflie 2.x, flowdeck v2, and a 2.0 multi-ranger deck

**Reply by ugrm** [reply]
I also tried running it via python3 (https://forum.bitcraze.io/viewtopic.php?t=5300#:~:text=crazyflie%20python%20lib%20repository) using the python lib library examples

**Reply by ugrm** [reply]
This is my log client after connecting it with the crazyradio 2.0
![image](https://github.com/bitcraze/discussions/assets/145661518/e316f1a5-2c6f-4add-83b3-9622c6ec8f1e)

**Reply by knmcguire** [reply]
Hi! the console tab would be more suitable as it will give us any error message when the initialization has gone wrong. Could you give that instead? The Log client tab is more for logging of the client  itself, not the crazyflie

**Reply by ugrm** [reply, marked as answer]
Thanks for the quick response.
I disconnected the crazyflie from the radio a few times and it just worked. I'll make sure to post the console tab in the future for problems. Thanks!
